Bottom: fce7f5b1ad9fc45b141a4b75dd97232f90c5a2a8
Top:    0d927919f345afa1a9fc1b3b09f751231c2bf751
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-27 04:55:16 +0100

Refresh of shiny-3d-eye-candy-0

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 46b368c..3a9f227 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -97,16 +97,16 @@ compute_depth (int group)
 {
   static int last_depth_computed = 0;
 
-  int bottom_group;
   int top_group;
+  int bottom_group;
   int min_copper_group;
   int max_copper_group;
   int num_copper_groups;
   int middle_copper_group;
   int depth;
 
-  bottom_group = GetLayerGroupNumberBySide (BOTTOM_SIDE);
   top_group = GetLayerGroupNumberBySide (TOP_SIDE);
+  bottom_group = GetLayerGroupNumberBySide (BOTTOM_SIDE);
 
   min_copper_group = MIN (bottom_group, top_group);
   max_copper_group = MAX (bottom_group, top_group);
